system-test/spanner.ts

```diff
@@ -4863,6 +4863,235 @@ describe('Spanner', () => {
           });
         });
 
+        describe('float32', () => {
+          const float32Query = (done, database, query, value) => {
```

Comment:
All the testcase are using float64 not float32. If param type is not passed then it take float64 as default. Add below for all tests
```
types: {
                v: 'float32',
              },
```

Reply:
Done